pegasus/emails/hoc_signup_2017_receipt_en.md

```diff
@@ -1,39 +1,47 @@
 
-You're making it possible for students all around the world to learn one Hour of Code that can change the rest of their lives, during December 4-10.
+Hour of Code runs December 4-10. We'll be in touch about new tutorials and other exciting updates as they come out. **In the meantime, what can you do now?**
 
-#### We'll be in touch about new tutorials and other exciting updates. What can you do now?
-
-## 1. Find a local volunteer to help you with your event.
+### 1. Find a local volunteer to help you with your event.
 [Search our volunteer map](https://code.org/volunteer/local) for volunteers who can visit your classroom or video chat remotely to inspire your students about the breadth of possibilities with computer science.
 
-## 2. Spread the word
-We need your help to reach organizers worldwide. Tell your friends about the #HourOfCode. [Use these helpful resources](https://<%= hostname %>/promote/resources) to promote your event.
+### 2. Spread the word & recruit your whole school
+We need your help to reach organizers worldwide. Tell your friends about the #HourOfCode. [Use these helpful resources](https://<%= hourofcode %>/promote/resources) to promote your event.
 
-## 3. Recruit your whole school for the Hour of Code
-[Send this email](https://<%= hostname %>/promote/resources#sample-emails) to your principal or [share these handouts](https://<%= hostname %>/promote/resources).
+Help recruit more people from your school and community by [sending our sample emails](https://<%= hourofcode %>/promote/resources#sample-emails) to your principal, a local group, or even some friends.
 
-## 4. Ask your employer to get involved
-[Send this email](https://<%= hostname %>/promote/resources#sample-emails) to your manager, or the CEO.
+### 3. Start planning your event
+Choose an [Hour of Code activity](https://<%= hourofcode %>/learn) for your classroom and [review this how-to guide](https://<%= hourofcode %>/us/how-to).
 
-## 5. Promote the Hour of Code in your community
-Recruit a local group or even some friends. [Send this email](https://<%= hostname %>/promote/resources#sample-emails).
+### From an Hour of Code to years of computer science
+<% if form.data["hoc_event_country_s"] == 'US' %>
+An Hour of Code is just the beginning. Whether you are an administrator, teacher, or advocate, we have [professional learning, curriculum, and resources to help you bring computer science classes to your school or expand your offerings.](https://<%= codedotorg %>/yourschool) If you already teach computer science, use these resources during CS Education Week to rally support from your administration, parents, and community.
```

Comment:
Do we want to link to `/yourschool/thanks` so they get the same resources but won't be confused by seeing the same form to fill out again?

Reply:
My bad it's `/yourschool/thankyou`. But I guess what's wrong with this is we don't yet put HOC census signups on the map on /yourschool yet right?

Reply:
Not sure. This is in the HoC signup email so they may not have filled out the census questions.

Reply:
I don't think we are putting the HOC Census signups on the map.

The top of the /yourschool page has the same Admin/Teacher/Parent resource links and the thank you page.

Reply:
Oh right! Let's keep it as `/yourschool` then. :)